src/blog/2024/01/dashboard-2-multi-user.md

```diff
@@ -0,0 +1,134 @@
+
+<iframe width="100%" height="100%" src="https://flows.nodered.org/flow/2fe8e6f1e7002f1ff6a9195ad1a153b6/share" allow="clipboard-read; clipboard-write" style="border: none; margin-bottom: 12px;"></iframe>
+
+Let's breakdown the above flow:
+
+1. We wire a `ui-event` node (which emits each time a user views a page) into a switch node
+2. Our switch node checks the `user.username` against a known list of admin users and branches "admin:" and "non-admin" users
+3. For admin users, the `change` node defines a message for our `ui-control` node to dynamically show content, in this case an "Admin" page, when appropriate.
+
+```json
+{
+    "pages":{
+        "show": ["Admin View"]
+    }
+}
+```
+
+All events going into `ui-control` are automatically filtered based on the `msg._client.user` object, so only the Admin users will receive the message to show the "Admin View" page, resulting in:
+
+![Showing Admin Only View](./images/multi-user-dashboard-admin.png "Showing Admin Only View"){data-zoomable}
+<figcaption class="-mt-6 mb-4 text-center"><b>The "Admin" view that is only made available to users registered as an "admin".</b></figcaption>
+
+Further extensions of this could also check `ui-event` in case a non-admin user tries to access the `/admin` page directly, in which case we can utilise `ui-control` to navigate the user away from the page immediately. See the [ui-control documentation](https://dashboard.flowfuse.com/nodes/widgets/ui-control.html#navigation) for more details on this.
+
+## Upcoming Webinar
+
+If you're interested in learning more about Dashboard 2.0 and in particular multi-user Dashboards, we're hosting a webinar on Thursday, 29th February. You can find out more information [here](/webinars/2024/node-red-dashboard-multi-user/)
+
+## Follow our Progress
+
+We aren't stopping here, we'll continue to push Dashboard 2.0 forward with future development, and you can track that progress on our GitHub Projects:
+
+- [Dashboard 2.0 Activity Tracker](https://github.com/orgs/FlowFuse/projects/15/views/1)
+- [Dashboard 2.0 Planning Board](https://github.com/orgs/FlowFuse/projects/15/views/4)
+- [Dashboard 1.0 Feature Parity Tracker](https://github.com/orgs/FlowFuse/projects/15/views/5)
+
+ If you have any feature requests, bugs/complaints or general feedback, please do reach out, and raise issues on our relevant [GitHub repository](https://github.com/FlowFuse/node-red-dashboard).
+
+ ## How to get started
+
+### FlowFuse Cloud
+The Personalised Multi-user Dashboard plugin is available in our Certified Nodes catalogue, accessible to our Teams and Enterprise customers. You just have to go to the Node-RED Palette Manager of your Node-RED instance and select the Certified Nodes catalogue. Every instance created from today onwards automatically comes with the necessary configuration. Already created instances need to be manually restarted.
```

Comment:
> Every instance created from today onwards automatically comes with the necessary configuration. Already created instances need to be manually restarted.

@MarianRaphael what's meant by this? Do users need to restart their instance in order for our plugin to show in the Certified Nodes library?